views/admin/exhibits/exhibit-metadata-form.php

```diff
@@ -63,6 +63,17 @@
                 <?php echo $this->formCheckbox('use_summary_page', $exhibit->use_summary_page, array(), array('1', '0')); ?>
             </div>
         </div>
+        <div id="cover-image-container" class="field">
+            <div class="two columns alpha">
+                <?php echo $this->formLabel('cover_image', __('Cover Image')); ?>
+            </div>
+            <div class="five columns omega inputs">
+                <p class="explanation">
+                  <?php echo __("Choose an item to represent this exhibit. Shown on the Browse Exhibits page and on the home page when the exhibit is featured."); ?>
+                </p>
+                <?php echo $this->partial('files/cover-image.php', array('file' => $exhibit->getFile())); ?>
```

Comment:
Using `getFile` here leads to the somewhat confusing situation that there appears to be a specifically selected image even when there's none and Omeka's just guessing the first image. It also means that saving the exhibit will inadvertantly "lock in" the first file at that moment as the "cover."

This should instead be specifically reading the `cover_image_file_id` only, so it accurately represents whether the user has picked a specific image or not. It may be convenient to add an accessor to the Exhibit for just getting *that* File or null (which would then be used here and within `getFile`)